source/_integrations/zwave_js.markdown

```diff
@@ -967,34 +967,42 @@ Zwavejs2Mqtt was renamed Z-Wave JS UI in September 2022. They are synonymous wit
 
 ### Can I switch between Z-Wave JS and Z-Wave JS UI?
 
-You can switch between the official Z-Wave JS add-on and the Z-Wave JS UI add-on. However, but you cannot run them both at the same time. Only one of them can be active at the same time.
+You can switch between the official Z-Wave JS add-on and the Z-Wave JS UI add-on. However, you cannot run them both at the same time. Only one of them can be active at the same time.
 
-### How to switch between Z-Wave JS and Z-Wave JS UI?
+### How to switch from Z-Wave JS to the Z-Wave JS UI add-on?
 
-To switch between the official Z-Wave JS add-on and the Z-Wave JS UI add-on, follow these steps:
+You can switch from the official **Z-Wave JS** add-on to the **Z-Wave JS UI** add-on. However, you cannot run them both at the same time. Only one of the add-ons can be active at the same time.
 
 Switching does not require renaming your devices.
 
-1. Disable the Z-Wave integration. **Do not remove the Z-Wave integration or you will lose all device and entity naming.** This will automatically stop the official Z-Wave JS add-on.
+1. Note your network security keys from the official add-on.
 
-2. Note your network security keys from the official add-on.
+2. Install and start the **Z-Wave JS UI** add-on.
+   - It may take a while for the add-on to start up.
 
-3. Install and start the Z-Wave JS UI add-on.
+3. Configure the **Z-Wave JS UI** add-on with the added control panel, including setting the location of your Z-Wave device and the network security keys.
+   - Open the **Z-Wave JS UI** web UI and go to **Settings** > **UI** > **Z-Wave**.
+   - Enter the security keys and region.
+   - Save your changes and in the top-right corner, select the {% icon "solar:restart-outline" %} restart button to reload the add-on configuration.
+   - Open the **Documentation** tab and copy the URL listed in the section **Setting up the Home Assistant Z-Wave JS integration**. You will need it in the next step.
 
-4. Configure the Z-Wave JS UI add-on with the added control panel, including setting the location of your Z-Wave device and the network security keys.
+4. Reconfigure the adapter.
+   - Select the **Z-Wave** integration and select the three-dot {% icon "mdi:dots-vertical" %} menu.
+   - From the menu, select **Reconfigure**, then **Reconfigure current adapter**.
+   - Uncheck the **Use the Z-Wave JS Supervisor add-on**.
+   - Under **WebSocket URL**, enter the URL you copied before.
+     - **Troubleshooting**: If it fails to connect although you pasted the right URL, check if you pasted all the security keys in the **Z-Wave JS UI** configuration.
 
-5. Add the Z-Wave integration again (even though it is still installed), and uncheck the "Use the Z-Wave JS Supervisor add-on". Enter the correct address for the community add-on in the URL field in the next step.
+5. Uninstall the official **Z-Wave JS** add-on.
```

Comment:
when uninstalling, should I delete all the related data? By default, this setting is disabled.